### DISCUSSION TITLE
[mpvnet] 修改下方进度条的样式

### DISCUSSION THREAD

**Discussion by jolyonaha** [hooke007/mpv_PlayKit#274] [libmpv]
![2023-02-15_15-38-35](https://user-images.githubusercontent.com/67818339/218964583-f52048c2-684d-4694-aa14-d07b44a532a5.png)
记得以前进度条中键有颗球作为标记，两边边框有圆角的
求教一下配置文件的路径以及大概的参数
谢谢作者

**Comment by hooke007** [comment]
参见 https://hooke007.github.io/unofficial/mpv_start.html#id10 osc.conf
